packages/aws-cdk/lib/cli/pretty-print-error.ts

```diff
@@ -9,7 +9,7 @@ export function prettyPrintError(error: unknown, debug = false) {
   if (err.cause) {
     const cause = ensureError(err.cause);
     console.error(chalk.yellow(cause.message));
-    printTrace(err, debug);
+    printTrace(cause, debug);
```

Comment:
Unrelated bug, but previously in dev mode we would print the stack trace of the parent error twice.